ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -36,6 +37,7 @@ component:
   guidevane: "Control vanes that meter refrigerant (on a centrifugal chiller) or air (on an AHU)."
   heater: "Component which provides heat to media."
   humidifier: "Component which humidifies."
+  light: "Artificial light source devices or natural illumination by capturing daylight."
```

Comment:
"Light" is not a thing or component, but a phenomenon. I think it is too generic to be included for a specific component.
It's best to use "lamp" for a light source and "daylight_device" or something more specific like "window", "reflector", etc. for a daylight component.

Reply:
@pisuke "lamp" is used as artificial light source devices, and description of daylight had been removed.